aliases/package-manager.aliases.bash

```diff
@@ -21,8 +21,15 @@ if _omb_util_binary_exists emerge; then
   alias esync="${_omb_tmp_sudo}emerge --sync" # Enoch SYNC
   alias eb="${_omb_tmp_sudo}ebuild" # Enoch Build
   alias er="${_omb_tmp_sudo}emerge -c" # Enoch Remove
+  alias emfu="${_omb_tmp_sudo}emerge --sync && ${_omb_tmp_sudo}emerge -uDN @world" # Enoch Upgrade System
+  # Deprecated, retained for backwards compatibility
```

Comment:
```suggestion
  # Deprecated, retained for backward compatibility
```

Reply:
I sometimes see "backwards compatibility" online, but "backwards" is technically an adverb but not an adjective.

Reply:
Can we also add a sentence "Use `cr` instead."?

Reply:
[wikipedia](https://en.wikipedia.org/wiki/Backward_compatibility) says:

> Backward compatibility (sometimes known as backwards compatibility) is a property of an operating system, software, real-world product, or technology that allows for interoperability with an older legacy system, or with input designed for such a system, especially in telecommunications and computing.

So this shouldn't be a typo. Do you still think it needs to be modified?

Reply:
I understand that you intentionally chose "backwards", and I wouldn't say this *needs* to be modified. However, at least, backward compatibility is grammatically *more* natural and seems to be more common. It's just preferences to some extent, but I'd like to make it sound more natural.

Reply:
For example, [‘Backward’ Versus ‘Backwards’ - Quick and Dirty Tips](https://www.quickanddirtytips.com/articles/backward-versus-backwards/#:~:text=I%20do%20have%20to%20say%20I%E2%80%99ve%20seen%20it%20quite%20a%20bit%20online%2C%20as%20in%20%E2%80%9Cbackwards%20compatibility%2C%E2%80%9D%20but%20that%E2%80%99s%20online.%20If%20you%20want%20to%20write%20properly%20for%20your%20job%20or%20for%20schoolwork%20or%20just%20for%20life%2C%20stick%20with%20%E2%80%9Cbackward%E2%80%9D%20for%20the%20adjective.) says *"If you want to write properly for your job or for schoolwork or just for life, stick with “backward” for the adjective.*".

Just to clarify, I'm not requesting you to use "backward compatibility" instead of "backwards compatibility" in your daily conversations or in your documents. However, when we collaborate, we need to find the point of compromise.